### PR TITLE
Add ultra icons

### PR DESCRIPTION
**Pull request by michelinewu**

### CODE REVIEW

**Review by gettinToasty — APPROVED** [review]

File: app/components-react/shared/UltraIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+        >
+          <stop stopColor="#2DE8B0" />
+          <stop offset="0.488027" stopColor="#CBE953" />
+          <stop offset="0.758621" stopColor="#FFAB48" />
+          <stop offset="1" stopColor="#FF5151" />
+        </linearGradient>
+      </defs>
+    </svg>
+  );
+}
```
Comment (gettinToasty) [inline app/components-react/shared/UltraIcon.tsx:201]: it looks like `simple` and the "default" icon type are two different svgs? is this intentional?

File: app/services/recent-events.ts [hunk 2]
```diff
@@ -171,12 +171,12 @@ const filterName = (key: string): string => {
     filter_subscription_9_months: $t('9 Months'),
     filter_subscription_12_months: $t('12 Months'),
     filter_subscription_minimum_enabled: $t('Min. Months'),
-    primesub: $t('Prime'),
+    primesub: $t('Ultra'),
```
Comment (gettinToasty) [inline app/services/recent-events.ts:174]: all of the replacements in this file actually refer to Twitch Prime subs, so they should remain as Prime

**Review by gettinToasty — CHANGES_REQUESTED** [review]
Couple of small things, including something I didn't catch in my first review

(Also thanks for fixing the tests for me :))

File: app/services/recent-events.ts (hunk 2 above)
Comment (michelinewu) [reply app/services/recent-events.ts:174]: Done.

File: app/services/recent-events.ts [hunk 3]
```diff
@@ -375,7 +375,7 @@ class RecentEventsViews extends ViewHandler<IRecentEventsState> {
       });
     }
     if (event.sub_type === 'primepaidupgrade') {
-      return $t('has converted from a Prime Gaming sub to a %{tier} sub', {
+      return $t('has converted from an Prime Gaming sub to a %{tier} sub', {
```
Comment (gettinToasty) [inline app/services/recent-events.ts:378]: grammar check kekw

**Review by gettinToasty — CHANGES_REQUESTED** [review]

File: app/services/recent-events.ts (hunk 3 above)
Comment (michelinewu) [reply app/services/recent-events.ts]: Fixed - thanks!

**Review by gettinToasty — APPROVED** [review]

**Pull request merged**